### ISSUE TITLE
DB versioning and schema update scripts

### ISSUE THREAD

**Issue by ekala** [ushahidi/SwiftRiver#172]
At the moment, all schema updates are made in the schema setup script `swiftriver.sql`. It would really be helpful to publish the DB upgrade scripts in addition to making changes in the main setup script.

For the core (Ushahidi), we maintain schema upgrade scripts in the `sql` folder and the naming convention is `upgrade<from> - <to>` where `from` and `to` represent the current and target schema versions respectively. I'm not sure whether this is the right approach but so far it's been working out just fine. Thoughts? @69mb @dkobia

**Comment by dkobia** [comment]
Wondering if xxxx1.sql, xxxx124.sql etc., might be a better approach?

**Comment by nlewis2142** [comment]
I've worked with several system in the past, the one I found that worked best was:

```
sql/
    create/
        [dbnamehere].sql
    updates/
        [dbnamehere]/
            yyyy_mm_dd_nn_TableNameOrSubject.sql
            2012_06_17_00_settings.sql
            2012_06_18_00_channel_filters.sql
            2012_06_18_01_users.sql
```

The main single-file SQL goes in create for anyone setting up the db. This file is always up to date. All other updates go in the updates/dbnamehere/ folder and are organized by date, meaning they will always be in order. The "nn" is for if multiple are submitted on the same day. Start at 00 and increment.

From there, it's fairly easy to batch apply SQL files with a simple shell script based on which version you're currently at.
